DigitalLearningSolutions.Web/Views/SuperAdmin/Faqs/SuperAdminFaqs.cshtml

```diff
@@ -0,0 +1,42 @@
+@using DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Faqs
+@model FaqsPageViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/superAdmin/faqs.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "FAQs";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-one-quarter">
+    <nav class="side-nav-menu" aria-label="Side navigation bar">
+      <partial name="~/Views/SuperAdmin/Shared/_SideNavMenu.cshtml" model="Model" />
+    </nav>
+  </div>
+
+  <div class="nhsuk-grid-column-three-quarters">
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-one-third">
+        <h1 id="page-heading" class="nhsuk-heading-xl">FAQs</h1>
+      </div>
+      <div class="nhsuk-grid-column-two-thirds heading-button-group">
+        <a class="nhsuk-button nhsuk-button--primary heading-button" role="button">
```

Comment:
Just remembered to put this comment down- there is no "nhsuk-button--primary" class so it can be removed safely. The primary button styling is all contained with nhsuk-button.